<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetAllForums</class>
 <widget class="QDialog" name="GetAllForums">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>331</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Get All Forums</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>11</y>
     <width>371</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Select Forum ID</string>
      </property>
      <widget class="QComboBox" name="Select_Forum_ID">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Select forum ID</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Name of Selected Forum</string>
      </property>
      <widget class="QLabel" name="Name_Forum">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>351</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Description of Selected Forum</string>
      </property>
      <widget class="QLabel" name="Description_Forum">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>351</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Close_Get_All_Forum">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
